yt/visualization/volume_rendering/render_source.py

```diff
@@ -290,7 +290,7 @@ def transfer_function(self, value):
         if not isinstance(value, valid_types):
             raise RuntimeError(
                 "transfer_function not a valid type, "
-                "received object of type %s" % type(value)
+                f"received object of type {type(value)}"
```

Comment:
> 
```suggestion
                f"received object of type {type(value)}."
```

Reply:
I'm trying not to change the result at all, so I'd rather skip this

Reply:
Fair enough, I'm happy with the PR as is.